src/viem/estimateFeesPerGas.ts

```diff
@@ -58,3 +63,28 @@ export async function getMaxPriorityFeePerGas(client: Client, feeCurrency?: Addr
   })
   return hexToBigInt(maxPriorityFeePerGasHex)
 }
+
+// Get gas price minimum with optional fee currency, this is Celo specific
+export async function getGasPriceMinimum(client: Client, feeCurrency: Address | undefined) {
```

Comment:
nit: Since this is `Celo` specific, how do feel about adding `Celo` to the method name?
Something like: `getCeloGasPriceMinimum`